### PR TITLE
Add sheikh Qahtani and Asmari Mokhtasar

### LINKED ISSUE #1786: Add new reciter: Sheikh Sabri Abdulhakm with Arabic commentary of Mukhtaser by D. Abdullah Al-Asmari
السلام عليكم ورحمة الله وبركاته 
I suggest to add Arabic commentary of Tafseer Mukhtaser by D. Abdullah Al-Asmari with recitation of Sheikh Sabri Abdulhakm recorded by Tafseer center for quaranic studies.
[Sample](https://download.quranicaudio.com/ayah/mokhtasar-asmari/001-1.mp3)

وجزاكم الله خيرا

### LINKED ISSUE #1793: Add new reciter: Sheikh Khaled Alqahtani
السلام عليكم ورحمة الله وبركاته 

I suggest to add Sheikh Khaled Al-qahtani to list of app reciters خالد القحطاني

https://download.quranicaudio.com/quran/khaalid_al-qahtaanee/
Timing database file:
[khalid_alqahtani.zip](https://github.com/quran/quran_android/files/7576017/khalid_alqahtani.zip)
وجزاكم الله خيرا

### PR DESCRIPTION
**Pull request by ahmedre**
Fixes #1786, and fixes #1793.

### CODE REVIEW

File: common/audio/src/main/res/values-fa/readers.xml [hunk 1]
```diff
@@ -67,4 +67,6 @@
   <string name="qari_ibrahim_alakhdar_gapless">ابراهیم الاخضر (بی‌وقفه)</string>
   <string name="qari_muaiqly_gapless">ماهر المعيلقي (بی‌وقفه)</string>
   <string name="qari_yasser_salama_hadr_gapless">یاسر سلامه (حدر، ی‌وقفه)</string>
+  <string name="qari_khalid_qahtani_gapless">خالد القحطانی (بی‌وقفه)</string>
+  <string name="qari_mokhtasar_asmari_gapless">عبدالله الأسمري (المختصر الصوتي, بی‌وقفه)</string>
```
Comment (benomaire) [inline common/audio/src/main/res/values-fa/readers.xml:71]: I like the other comma `،` for rtl languages

File: common/audio/src/main/res/values-fa/readers.xml [hunk 2]
```diff
@@ -67,4 +67,6 @@
   <string name="qari_ibrahim_alakhdar_gapless">ابراهیم الاخضر (بی‌وقفه)</string>
   <string name="qari_muaiqly_gapless">ماهر المعيلقي (بی‌وقفه)</string>
```
Comment (benomaire) [inline common/audio/src/main/res/values-fa/readers.xml:68]: seems like ل and ق are swapped.

File: common/audio/src/main/res/values-ku/readers.xml [hunk 3]
```diff
@@ -67,4 +67,6 @@
   <string name="qari_ibrahim_alakhdar_gapless">Îbrahîm El-Axdar (مورەتەل)</string>
   <string name="qari_muaiqly_gapless">ماھیر موعەیقلی (مورەتەل)</string>
   <string name="qari_yasser_salama_hadr_gapless">Yasser Salama (حدر، مورەتەل)</string>
+  <string name="qari_khalid_qahtani_gapless">Xalid El Qehtanî (مورەتەل)</string>
+  <string name="qari_mokhtasar_asmari_gapless">Abdullah Al-Asmari (Tefsîra erebî, مورەتەل)</string>
```
Comment (benomaire) [inline common/audio/src/main/res/values-ku/readers.xml:71]: This language is becoming a mess :)
I mean mixing two alphabets

**Comment by ahmedre** [comment]
جزاكم الله خيراً @benomaire made the changes

**Pull request merged**